plugins/metadata/MetadataBulkUploadXmlPlugin.php

```diff
@@ -190,6 +190,61 @@ public static function contributeToSchema($type)
 			</xs:appinfo>
 		</xs:annotation>
 	</xs:element>
+	
+	<xs:complexType name="T_metadataReplacementOptions">
+		<xs:sequence>
+			<xs:element name="shouldTransferMetadata" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:appinfo>
+						<example>
+							<item>
+								<pluginReplacementOptions>
+									...
+									...
+									...
+								</pluginReplacementOptions>
+							</item>
+						</example>
+					</xs:appinfo>
+				</xs:annotation>
+				<xs:simpleType>
+					<xs:restriction base="xs:string">
+						<xs:enumeration value="true"/>
+						<xs:enumeration value="false"/>
+					</xs:restriction>
+				</xs:simpleType>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	
+	<xs:complexType name="T_pluginReplacementOptions">
```

Comment:
T_pluginReplacementOptions should be defined in the code XSD while T_metadataReplacementOptions should be contributed here by the metadata plugin.
Use inheritance to achieve that, see scene as example.